### ISSUE TITLE
Does it work?

### ISSUE THREAD

**Issue by alecapu** [gvsumasl/EchoprintForAndroid#11]
Hi,
I'm very interested in your EchoprintForAndroid. I'm tryng it using EchoPrintServer as server backend (official python version and nodejs porting) but I'm not able to match any songs... 
Witch server implementation do you use? Can you help me?

**Comment by alexrestrepo** [comment]
Are you using the latest backend api? I believe it has changed a couple of times since the code was posted.

**Comment by alecapu** [comment]
Yes the latest version. I've added the version on the url parameters. Every
thing works fine but.... it doesn't match any songs in my db.... :-(
Very strange...
Il 26/Gen/2016 06:14, "Alex Restrepo" notifications@github.com ha scritto:

**Comment by fillobotto** [comment]
I confirm it's still working with [this porting](https://github.com/jhurliman/node-echoprint-server)
